cli/execute.ts

```diff
@@ -44,7 +47,9 @@ export async function handleExecute({
   });
   const manifest = await importManifest(bundlePath);
   if (manifest.networkDomains && manifest.networkDomains.length > 1 && !allowMultipleNetworkDomains) {
-    return printAndExit('Using multiple network domains requires approval from Coda. Visit https://coda.io/packs/build/latest/support#approvals to make a request. Disable this warning by including the flag: --allowMultipleNetworkDomains');
+    return printAndExit(
+      'Using multiple network domains requires approval from Coda. Visit https://coda.io/packs/build/latest/support#approvals to make a request. Disable this warning by including the flag: --allowMultipleNetworkDomains',
```

Comment:
Ah, I guess we have enforced standardized `prettier` formatting in the `packs` repo now, but still not here... For later.